rubocop.yml

```diff
@@ -50,6 +49,12 @@ Style/AccessorMethodName:
   Enabled: false
 Style/Alias:
   Enabled: false
+Style/BlockDelimiters:
+  EnforcedStyle: semantic
```

Comment:
I mostly also agree with `line_count_based` approach and remember that we might have it even written in code style guides. Or had it written there. But there is one potential issue. 

In Rspec with `expect... to change...` you regardless of number of lines you need to use { ... } cause of how ruby evaluates do...end and {....} Example:

```
>> def with(arg=nil, &block)
|    block.call("called: #{arg.inspect}") if block_given?
|    p "return: #{arg.inspect}"
|    end #=> :with
>> with with(42) { |x| p x } #=> "return: \"return: 42\""
"called: 42"
"return: 42"
"return: \"return: 42\""
>> with with(42) do |x| p x end #=> "return: \"return: 42\""
"return: 42"
"called: \"return: 42\""
"return: \"return: 42\""
```

So for Ruby do...end is more like: `with(with(42)) do |x| p x end`
and `{...}` is more like: `with(with(42) { |x| p x })` and Rspec takes advantage of it with its DSL.